src/aind_metadata_mapper/ephys/camstim_session.py

```diff
@@ -0,0 +1,469 @@
+"""
+File containing CamstimSession class
+"""
+
+import argparse
+import datetime
+import io
+import json
+from pathlib import Path
+
+import aind_data_schema
+import aind_data_schema.core.session as session_schema
+import np_session
+import npc_ephys
+import npc_mvr
+import npc_session
+import npc_sessions
+import npc_sync
```

Comment:
this is a lot of dependencies - is there code that it makes sense to migrate here instead?

Reply:
I've removed the dependency on npc_session. 
npc_ephys, npc_mvr, npc_sync all perform some sort of sync-timing operations that I think would not be worth trying to replicate in this repo, as they are rather elaborate (I don't understand them). Perhaps a discussion about architecture is necessary?